starter-code/scripts/article.js

```diff
@@ -34,29 +34,38 @@ Article.prototype.toHtml = function() {
 // and use it to instantiate all the articles. This code is moved from elsewhere, and
 // encapsulated in a simply-named function for clarity.
 Article.loadAll = function(rawData) {
-  rawData.sort(function(a,b) {
-    return (new Date(b.publishedOn)) - (new Date(a.publishedOn));
+ rawData.sort(function(a,b) {
+ return (new Date(b.publishedOn)) - (new Date(a.publishedOn));
   });
 
   rawData.forEach(function(ele) {
     Article.all.push(new Article(ele));
   })
 }
-
 // This function will retrieve the data from either a local or remote source,
 // and process it, then hand off control to the View.
 Article.fetchAll = function() {
   if (localStorage.rawData) {
     // When rawData is already in localStorage,
     // we can load it with the .loadAll function above,
     // and then render the index page (using the proper method on the articleView object).
-    Article.loadAll(); //TODO: What do we pass in to loadAll()?
+    Article.loadAll(JSON.parse(localStorage.rawData)); //TODO: What do we pass in to loadAll()?
```

Comment:
Note this also works as well : Article.loadAll(JSON.parse(localStorage.getItem('rawData')));